src/Components/CommentCard.jsx

```diff
@@ -1,16 +1,66 @@
-import axios from "axios"
-import { useState } from "react"
+import axios from "axios";
+import { useState } from "react";
 
-function CommentCard({comment}){
-    const [vote, setVotes] = useState(comment.votes)
-    return(
-         <>
-             <h3>{comment.author}</h3>
-             <p>{comment.created_at}</p>
-             <p className="comment-text">{comment.body}</p>
-             <p className="votes">Votes: {vote}</p>
-         </>
-    )
+function CommentCard({ comment }) {
+  const [vote, setVotes] = useState(comment.votes);
+  const voteTracker = {
+    inc_votes: 1,
+  };
+
+  function addVote() {
+    voteTracker.inc_votes = 1
+  }
+  function minusVote() {
+    voteTracker.inc_votes = -1
+  }
+
+  function modifyVotes() {
+    axios
+      .patch(
+        `https://nc-news-jstr.onrender.com/api/comments/${comment.comment_id}`,
+        voteTracker
+      )
+      .then((response) => {
+        console.log(response);
+      })
+      .catch((err) => {
+        alert(err)
+        console.log(err);
```

Comment:
you're successfully catching this error, but you're not doing anything with it (other than alerting the user something went wrong). What this means is while I'm offline, I'm voting on comments and your app is telling me that the votes are increasing. In the database, the votes aren't changing. How can you give the user more comprehensive feedback that their vote didn't go through?